source/install/self-managed-changelog.md

```diff
@@ -5,12 +5,268 @@ This changelog summarizes updates to [Mattermost Team Edition](https://mattermos
+ - Fixed an issue with Collapsed Reply Threads (Beta) where replying to a Thread caused users to re-follow the previously unfollowed Thread.
+ - Fixed an issue where floating timestamps appeared incorrectly on the right-hand side with Collapsed Reply Threads (Beta) enabled.
+ - Fixed an issue where pinned and saved posts were no longer highlighted.
+ - Disabled admin support email status check job on server startup.
+ - Fixed an issue on joining a missing channel as a System Admin.
+ - Fixed import process for imports with attachments.
+ - Fixed an error with app locations and binding filtering.
+
+### config.json
+Multiple setting options were added to ``config.json``. Below is a list of the additions and their default values on install. The settings can be modified in ``config.json``, or the System Console when available.
+
+#### Changes to Team Edition and Enterprise Edition:
+ - Under ``ServiceSettings`` in ``config.json``:
+    - Added ``EnableOnboardingFlow``, for enhanced user onboarding experience feature.
+    - Added ``EnablePermalinkPreviews`` to enable permalink previews.
+ - Under ``FileSettings`` in ``config.json``:
+    - Added ``MaxImageResolution`` config setting to control the maximum dimension (in pixels) of image uploads.
+ - Removed all of the following configs:
+   - ``EnableOnlyAdminIntegrations``
+   - ``RestrictCustomEmojiCreation``
+   - ``RestrictPostDelete``
+   - ``AllowEditPost``
+   - ``ImageProxyType``
+   - ``ImageProxyURL``
+   - ``ImageProxyOptions``
+   - ``UseExperimentalGossip``
+   - ``EnableTeamCreation``
+   - ``RestrictTeamInvite``
+   - ``RestrictPublicChannelManagement``
+   - ``RestrictPrivateChannelManagement``
+   - ``RestrictPublicChannelCreation``
+   - ``RestrictPrivateChannelCreation``
+   - ``RestrictPublicChannelDeletion``
+   - ``RestrictPrivateChannelDeletion``
+   - ``RestrictPrivateChannelManageMembers``
+   - ``DisableLegacyMFAEndpoint``
+   - ``ExperimentalTownSquareIsReadOnly``
+   - ``ExperimentalHideTownSquareinLHS``
+   - ``EnableXToLeaveChannelsFromLHS``
+   - ``CloseUnusedDirectMessages``
+   - ``ExperimentalChannelOrganization``
+   - ``ExperimentalChannelSidebarOrganization``
+   - ``EnableLegacySidebar``
+   - The legacy MFA endpoint
+   - ``utils/authorization.go`` and moved any permissions to the ``MakeDefaultRoles()`` function.
+
+### Database Changes
+ - Removed deprecated ``model.ComparePassword`` method.
+ - Removed deprecated ``Context.SourcePluginId`` field.
+ - Removed ``(*model.Client4).CheckUserMfa``.
+ - Removed ``(*model.Client4).GetServerBusyExpires``.
+ - Removed MB constant from model package.
+ - Removed use of pointers to the following types: 
+   - ``model.ChannelList``
+   - ``model.ChannelListWithTeamData``
+   - ``model.ChannelMembers``
+   - ``model.Preferences``
+   - ``model.ProductNotices``
+ - Renamed ``plugin.Context.IpAddress`` to ``plugin.Context.IPAddress``.
+ - Renamed fields in the model package to have more idiomatic names.
```

Comment:
These aren't Database changes. A new category `Library Changes` just for this release would be the best fit.